packages/flutter/lib/src/material/tabs.dart

```diff
@@ -915,6 +915,7 @@ class _TabBarViewState extends State<TabBarView> {
 ///
 /// Used by [TabPageSelector] to indicate the selected page.
 class TabPageSelectorIndicator extends StatelessWidget {
+  /// Creates an indicator used by [TabPageSelector].
```

Comment:
btw we should consider replacing this with a selector that doesn't need a tab controller and just reads out the page information from the nearest PageView.